package.json

```diff
@@ -6,7 +6,10 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "test": "jest"
+    "test": "jest",
+    "cy:run": "npx cypress run",
+    "cy:open": "npx cypress open",
+    "e2e:test": "npx browserslist@latest --update-db && start-server-and-test start http://localhost:3000 cy:run"
```

Comment:
Runs the e2e suite in one-go, starts a server off the build directory and asserts e2e functionality, hence why start-server-and-test is added, and also I usually update the browserlist in the same task